src/styles/theme.css

```diff
@@ -1,8 +1,29 @@
-body {
-  color: green;
+@font-face {
+  font-family: "Geist-Sans";
+  src: url('../../node_modules/geist/dist/fonts/geist-sans/Geist-Regular.woff2') format("truetype");
 }
 
-h1, h2, h3, h4, h5, h6 {
-  text-align: center;
-  color: black;
+@font-face {
+  font-family: "Geist-Mono";
+  src: url('../../node_modules/geist/dist/fonts/geist-mono/GeistMono-Regular.woff2') format("truetype");
+}
+
+:root {
+  --color-primary: #016341;
+  --color-secondary: #0E0F0C;
+  --color-tertiary: #151818;
+  --color-accent: #C3DDBA;
+  --color-white: #FFFFFF;
+  --color-black: #020202;
+  --color-gray: #F5F5F5;
+  --color-gray-background: #EDEFEC;
+  --font-primary: 'Geist-Sans', system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;;
+  --font-secondary: 'Geist-Mono', monospace;
+  --font-size-base: 16px;
+  --spacing-unit: 8px;
+}
+
+* {
+  font-family: var(--font-primary);
+  font-size: var(--font-size-base);
```

Comment:
Perhaps we should also add `box-sizing` too?
https://css-tricks.com/almanac/properties/b/box-sizing/

Reply:
Added!